src/js/containers/review/ReviewDataContainer.jsx

```diff
@@ -8,7 +8,6 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
 import * as ReviewHelper from '../../helpers/reviewHelper.js';
-import * as AgencyHelper from '../../helpers/agencyHelper.js';
```

Comment:
Not quite -- `AgencyHelper` did an AJAX call to grab a list of agency names and ids and then use the submission's `cgac_code` to look up an agency name. As the submission now includes the agency name directly, there's no need for the AJAX step.